src/locale/locales/ja/messages.po

```diff
@@ -17,6 +17,10 @@ msgstr ""
 msgid "(no email)"
 msgstr "メールがありません"
 
+#: src/screens/Messages/Conversation/index.tsx:135
+msgid "{0}"
+msgstr "{0}"
```

Comment:
This msgid/msgstr can be removed if https://github.com/bluesky-social/social-app/issues/3787 is resolved.